app/serializers/topic_list_item_serializer.rb

```diff
@@ -86,4 +87,12 @@ def include_featured_link_root_domain?
     SiteSetting.topic_featured_link_enabled && object.featured_link.present?
   end
 
+  def allowed_user_count
+    return object.allowed_users.count
```

Comment:
You don't need the `return` here since it's the last statement of the method.

(Also, weird that rubocop didn't catch it)

Reply:
Thanks, I added the Rubocop role for it now in https://github.com/discourse/discourse/commit/be15abcf7f40ca01d8b9dfe57a0c3bd8ed9fe8d9